tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -9,7 +9,7 @@
 
 class Settings(Base):
 
-    name = 'Settings'
+    origin = 'app://settings.gaiamobile.org'
```

Comment:
You can change the name property, but then you need to make sure that there are no other consumers of this anymore. 
I suspect this is namely used at: http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/accessibility/system/test_a11y_utility_tray_settings.py#26
So you would need to change it there too.